python/services/github_service.py

```diff
@@ -199,6 +199,34 @@ def add_user_to_team(self, user_name: str, team_id: int) -> None:
         self.github_client_core_api.get_organization(
             self.organisation_name).get_team(team_id).add_membership(user)
 
+    def add_all_users_to_team(self, team_name: str) -> None:
```

Comment:
For awareness, this is the kind of public method that should exist in this Class

All the small methods that essentially talk to the client should be made private or moved the Facade style class. But one battle at a time 😅